src/FINDOLOGIC/Export/Helpers/DataHelper.php

```diff
@@ -10,6 +10,14 @@ public function __construct($message = 'Empty values are not allowed!')
     }
 }
 
+class ValueIsNotUrlException extends \RuntimeException
```

Comment:
Better name would be something like `InvalidUrlException`.

Reply:
Done